From d92c7dbc36db08a503d73cafe6076207f1c0aab7 Mon Sep 17 00:00:00 2001
From: "dnyaneshwarx.r.jadhav@intel.com" <dnyaneshwarx.r.jadhav@intel.com>
Date: Wed, 9 Dec 2015 19:09:34 +0530
Subject: [PATCH 3/4] ASoC: Intel: sst: SSP BT A2DP configuration

The TX pin signals are driven at rising edge and sampled at falling edge.

Change-Id: Iece24dbccd673461d1edad91c0d5e93427850080
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-6123
Signed-off-by: dnyaneshwarx.r.jadhav@intel.com <dnyaneshwarx.r.jadhav@intel.com>
Reviewed-on: https://android.intel.com:443/446282
---
 sound/soc/intel/sst/sst_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/intel/sst/sst_platform.c b/sound/soc/intel/sst/sst_platform.c
index 607910a..cee8a2b 100644
--- a/sound/soc/intel/sst/sst_platform.c
+++ b/sound/soc/intel/sst/sst_platform.c
@@ -232,6 +232,7 @@ sst_ssp_configs_mrfld[SST_NUM_SSPS][SST_MAX_SSP_MUX][SST_MAX_SSP_DOMAINS] = {
 				.fs_frequency = SSP_FS_48_KHZ,
 				.active_slot_map = 0x3,
 				.start_delay = 1,
+				.data_polarity = 1,
 			},
 
 		},
-- 
1.9.1

